docs/docusaurus.config.js

```diff
@@ -1,88 +1,158 @@
   themeConfig: {
+    algolia: {
+      // this is configured via DocSearch here:
+      // https://github.com/algolia/docsearch-configs/blob/master/configs/rasa.json
+      disabled: true,
+      apiKey: '25626fae796133dc1e734c6bcaaeac3c', // FIXME: replace with values from our own index
+      indexName: 'rasa',
+      inputSelector: '.search-bar',
+      searchParameters: {
+        'facetFilters': ["tags:rasa-action-server"]
+      }
+    },
     navbar: {
-      title: 'Action Server',
-      logo: {
-        alt: 'Rasa',
-        src: 'https://rasa.com/static/60e441f8eadef13bea0cc790c8cf188b/rasa-logo.svg',
-      },
+      hideOnScroll: false,
+      title: 'Rasa Action Server',
       items: [
         {
-          label: 'Docs',
-          to: '/', // "fake" link
+          label: 'Rasa Open Source',
+          href: `${RASA_OPEN_SOURCE_SWAP_URL}/docs/rasa/`,
           position: 'left',
-          items: versions.length > 0 ? [
+        },
+        {
+          label: 'Rasa X',
+          position: 'left',
+          href: `${RASA_X_SWAP_URL}/docs/rasa-x/`,
+          target: '_self',
+        },
+        {
+          label: 'Rasa Action Server',
+          position: 'left',
+          to: path.join('/', BASE_URL),
+        },
+        {
+          href: 'https://github.com/rasahq/rasa',
+          className: 'header-github-link',
+          'aria-label': 'GitHub repository',
+          position: 'right',
+        },
+        {
+          target: '_self',
+          href: 'https://blog.rasa.com/',
+          label: 'Blog',
+          position: 'right',
+        },
+        {
+          label: 'Community',
+          position: 'right',
+          items: [
+            {
+              target: '_self',
+              href: 'https://rasa.com/community/join/',
+              label: 'Community Hub',
+            },
+            {
+              target: '_self',
+              href: 'https://forum.rasa.com',
+              label: 'Forum',
+            },
             {
-              label: versions[0],
-              to: '/',
-              activeBaseRegex: versions[0],
+              target: '_self',
+              href: 'https://rasa.com/community/contribute/',
+              label: 'How to Contribute',
             },
-            ...versions.slice(1).map((version) => ({
-              label: version,
-              to: `${version}/`,
-              activeBaseRegex: version,
-            })),
-            ...legacyVersions,
-          ] : [
             {
-              label: 'Latest',
-              to: '/',
-              activeBaseRegex: `/`,
+              target: '_self',
+              href: 'https://rasa.com/showcase/',
+              label: 'Community Showcase',
             },
-            ...legacyVersions,
           ],
         },
       ],
     },
     footer: {
-      style: 'dark',
       copyright: `Copyright © ${new Date().getFullYear()} Rasa Technologies GmbH`,
     },
     gtm: {
       containerID: 'GTM-PK448GB',
```

Comment:
This is out of date, it should be `containerID: 'GTM-MMHSZCS',`

Reply:
7af9f15